statsmodels/regression/tests/test_tools.py

```diff
@@ -51,12 +51,12 @@ def test_inf_nan(self, bad_value):
             _MinimalWLS(endog, self.exog1, self.weights1,
                         check_endog=True, check_weights=True).fit()
         assert err.type is ValueError
-        assert 'endog' in str(err)
+        assert 'endog' in str(err.value)
```

Comment:
Would be better to use match as part of pytest.raises

Reply:
done.